src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -154,6 +155,18 @@ interface LicenceRepository :
   )
   fun getDraftLicencesPassedReleaseDate(): List<Licence>
 
+  @Query(
+    """
+    SELECT *
+        FROM licence
+        WHERE kind = 'HDC'
+        AND status_code IN ('IN_PROGRESS', 'SUBMITTED', 'APPROVED')
+        AND conditional_release_date - (INTERVAL '9' DAY) <= CURRENT_DATE;
+    """,
+    nativeQuery = true,
+  )
+  fun getDraftHdcLicencesPassedReleaseDate(): List<HdcLicence>
```

Comment:
I'm not sure how I personally feel about this name
It's technically true, but it works differently to `getDraftLicencesPassedReleaseDate` for two reasons:
- This includes `APPROVED` licences
- More importantly, this isn't checking against the LSD

I'm not sure if a name referencing that these licences are being deactivated because they're no longer suitable for HDC release would be more appropriate? Happy to discuss! It's also a pretty minor thing at the end of the day

Reply:
I dislike naming things so name wise, that sounds cool. Changed to `getDraftLicencesIneligibleForHdcRelease`